android/gradle.properties

```diff
@@ -2,3 +2,7 @@ org.gradle.jvmargs=-Xmx1536M -XX:+UseParallelGC
 android.useAndroidX=true
 android.enableJetifier=true
 extra-gen-snapshot-options=--obfuscate
+org.gradle.daemon=true
+org.gradle.parallel=true
+org.gradle.configureondemand=true
+org.gradle.caching=true
```

Comment:
those are to improve compilation time for android?

Reply:
Yes, exactly